target_chains/ethereum/contracts/Deploying.md

```diff
@@ -63,12 +45,10 @@ This is the deployment process:
 4. Prepare the required keys for deployment. You can find more information about them on notion. Then:
    - Export the secret recovery phrase for the deployment account. Please store it in a file and read
      the file into `MNEMONIC` environment variable like so: `export MNEMONIC=$(cat path/to/mnemonic)`.
-   - If you are modifying an existing contract, make sure that the multisig-cli has the operational
-     key stored in `governance/multisig_wh_message_builder/keys/key.json`.
-   - export the Infura RPC API key to `INFURA_KEY` if you are deploying to a network that uses an Infura RPC.
 5. Make sure the deployment account has proper balance on this network and top it up if needed. Search
    for testnet faucets if it is a testnet network. Sometimes you need to bridge the network token (e.g., L2s).
-6. Deploy the new contract or changes using the [`deploy.sh`](./deploy.sh) script. If you have made changes
+6. Add the required chain configuration in the contract manager file [EvmChains.yaml](../../../contract_manager/store/chains/EvmChains.yaml)
+7. Deploy the new contract or changes using the [`deploy.sh`](./deploy.sh) script. If you have made changes
    to [`chains.ts`](../../../governance/xc_admin/packages/xc_admin_common/src/chains.ts), please make sure to
    run `npx lerna run build --scope="@pythnetwork/pyth-evm-contract" --include-dependencies` in the
```

Comment:
Can we add this command to the `deploy.sh` script and remove the instructions here? i think it'll make it simpler